template/src/routes/__tests__/__snapshots__/app.component.spec.js.snap

```diff
@@ -1,11 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Why are all these snapshots removed?

Reply:
I thought we want to remove all snapshots from tamplate. It's not necessary to keep them here but if you want then I will get them back

Reply:
I would keep them so if you install the project and run `yarn test` all tests should pass by defaulut (including snapshots)